core/templates/components/rubrics-editor/rubrics-editor.component.html

```diff
@@ -55,6 +55,8 @@
           <div>
             <button type="button"
                     class="btn btn-danger protractor-test-delete-rubric-explanation-button"
+                    [disabled]="!this.isMediumLevelExplanationValid(this.rubric.getDifficulty(), this.editableExplanations[this.rubric.getDifficulty()][idx])
+                      || this.ismediumLevelExplanationLengthEqualsOne(this.rubric.getDifficulty())"
```

Comment:
If we keep this format, it's easier to write this as `[disabled]="!this.isMediumLevelExplanationValid() || this.ismediumLevelExplanationLengthEqualsOne()"` and then retrieve the arguments from within those functions. (Rather than retrieving them inline here and passing them in as an argument.

Reply:
Also, I think we should allow people to delete even if something is invalid.

Reply:
The reason that I'm passing the argument is to get the current data that the user is entering so that I can check if its length is greater than 0 or not, I'm not sure if I can get the live data directly in the component file.
Yeah, you are right that people should be able to delete it but the reason I'm disabling it is that way it will be mandatory to have one explanation, and if they don't like the one that comes by default they can replace it.

Reply:
If we do not allow the user to delete, then we should show text that explains why the user can't delete the Medium rubric.

Reply:
done